packages/common/src/api/comments.ts

```diff
@@ -18,12 +18,13 @@ const optimisticUpdateCommentList = (
   updateRecipe: (prevState: Comment[] | undefined) => void, // Could also return Comment[] but its easier to modify the prevState proxy array directly
   dispatch: ThunkDispatch<any, any, any>,
   userId?: number,
-  page: number = 0
+  page: number = 0,
+  sortMethod: TrackCommentsSortMethodEnum = TrackCommentsSortMethodEnum.Top
```

Comment:
Would we have different sort methods for other comments or could CommentsSortMethodEnum suffice instead of TrackCommentsSortMethodEnum?

Reply:
Yeah we have track timestamp as a unique one